src/main/java/com/google/sps/data/BackgroundTaskManager.java

```diff
@@ -26,8 +30,21 @@ public void replaceFaultyInstances() {
     }
 
     @Override
-    public void deleteInactiveAttendees() {
-        throw new RuntimeException("Unimplemented");
+    public void deleteInactiveAttendees(DatastoreClientInterface datastore, String sessionId) { 
+        List<Attendee> attendees = datastore.getAttendeesInSession(sessionId);
+        for(int i = 0; i <= attendees.length(); i++){
+            AttendeeInterface currentAttendee = attendees.get(i);
+            AttendeeeInterface screenName = currentAttendee.getScreenName();
+            long timePolled = currentAttendee.getTimePolled();
+            long currentTime = System.currentMillis(); 
+            long twoMins = 120000;
```

Comment:
Create a constant at the top of the class:

// If an attendee last polled the server longer this amount of time ago, we assume the attendee
// left and we delete them from the database. < --- use appropriate javadoc format for this comment
private static final LAST_TIME_POLLED_DEADLINE_MS = 2 * 60 * 1000;  // 2 minutes